### ISSUE TITLE
Release runtime api version 3

### ISSUE THREAD

**Issue by eskimor** [paritytech/polkadot#6292]
So we stabilize the disputes() function.

**Comment by eskimor** [comment]
Added to disputes board, as this enables new dispute chain import.

**Comment by eskimor** [comment]
@tdimitrov can we do this together with the execution environment?

**Comment by tdimitrov** [comment]
> @tdimitrov can we do this together with the execution environment?

Yes. [This change](https://github.com/paritytech/polkadot/pull/6161/files#diff-ea8693be4644e18e096279d95fb9277c104382a3ce7c473d388c04be9facfc9eR122) does it more or less.

**Comment by eskimor** [comment]
@tdimitrov execution environment will be merged this week - anything needs to be done for this one, or is merging that PR enough?

**Comment by tdimitrov** [comment]
I've left some comments in https://github.com/paritytech/polkadot/pull/6161 that should be addressed before the merge. Nothing major - missing implementation of `disputes()` in polkadot and kusama runtimes.

Fixing them and a test on versi should be enough to release v3.

**Comment by eskimor** [comment]
Let's release this asap - the recent incident on Kusama would likely not even have been noticed with this in place. @tdimitrov  @s0me0ne-unkn0wn - please let's get this into the next release.

**Comment by sandreim** [comment]
> Let's release this asap - the recent incident on Kusama would likely not even have been noticed with this in place. 

Can you detail a bit why the issue on Kusama would have been avoided ?

**Comment by eskimor** [comment]
I forgot that those redundant statements are still filtered out in block creation, so it would not have helped actually.
